--- github/IssueComment.py	2026-05-19 15:39:59.104386445 +0200
+++ github/IssueComment.py	2026-05-19 15:43:54.135290941 +0200
@@ -24,11 +24,12 @@
 from typing import Any, TYPE_CHECKING
 from datetime import datetime, timezone
 
+import github.NamedUser
 from github.GithubObject import CompletableGithubObject
 from github.GithubObject import Attribute, NotSet
 
 if TYPE_CHECKING:
-    pass
+    from github.NamedUser import NamedUser
 
 
 class IssueComment(CompletableGithubObject):
@@ -59,7 +60,7 @@
         self._reactions: Attribute[dict[str, Any]] = NotSet
         self._updated_at: Attribute[datetime] = NotSet
         self._url: Attribute[str] = NotSet
-        self._user: Attribute[dict[str, Any]] = NotSet
+        self._user: Attribute[NamedUser] = NotSet
 
     def __repr__(self) -> str:
         # TODO: replace "some_attribute" with uniquely identifying attributes in the dict, then run:
@@ -131,7 +132,7 @@
         return self._url.value
 
     @property
-    def user(self) -> dict[str, Any]:
+    def user(self) -> NamedUser:
         self._completeIfNotSet(self._user)
         return self._user.value
 
@@ -164,5 +165,7 @@
         if "url" in attributes:  # pragma no branch
             self._url = self._makeStringAttribute(attributes["url"])
         if "user" in attributes:  # pragma no branch
-            self._user = self._makeDictAttribute(attributes["user"])
+            self._user = self._makeClassAttribute(
+                github.NamedUser.NamedUser, attributes["user"]
+            )
 
